dashboard/app/templates.html

```diff
@@ -127,6 +128,7 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 					{{formatLateness $.Now $b.ReportedTime}}
 				{{end}}
 			</td>
+                        <td class="stat">last activity {{formatLateness $.Now $b.LastActivity}}</td>
```

Comment:
Did you run tests? If not you need to run them. I think this will crash.

Reply:
Yes I ran the presubmit tests as well as `go test ./dashboard/app`. Should I run other tests as well? It seems that `go test ./dashboard/app` uses cached results even if I make drastic changes to templates.html

Reply:
Humm... interesting. Never spotted this issue before... maybe I was lucky.
I would then do some simple no-op change to source files before running tests.

Reply:
You were absolutely right, once I changed a .go source file and re-triggered the test, it wouldn't complete. I believe it should be fixed now.

Reply:
Digging into the test caching issue, go command is actually smart enough to log all files opened by the test to reset cache on any changes:
https://go-review.googlesource.com/c/go/+/81895/
But the problem is that we open the *.html files in the init() function (actually global var initialization), and testing package starts logging opened files only from main. This is https://github.com/golang/go/issues/44625